docs/deployments/system-packages.md

```diff
@@ -50,7 +50,7 @@ Cortex's base Docker images are listed below. Depending on the Cortex Predictor
 
 <!-- CORTEX_VERSION_BRANCH_STABLE x6 -->
 * Python Predictor (CPU): `cortexlabs/python-predictor-cpu-slim:master`
-* Python Predictor (GPU): `cortexlabs/python-predictor-gpu-slim:master`
+* Python Predictor (GPU): `cortexlabs/python-predictor-gpu-slim:master-cuda10.1` (also available in cuda10.0, cuda10.2, and cuda11.0)
```

Comment:
I updated these docs, let me know if it looks ok to you

Reply:
One thing to keep in mind: if we get any users in the future asking about the CUDNN versions, then at that point, it may be necessary to revisit this. So for now, I think it's okay the way it is.